### DISCUSSION TITLE
webgazer.js is based on which release of Webgazer?

### DISCUSSION THREAD

**Discussion by brkfst42** [jspsych/jsPsych#2856] [Q&A]
Hi the [Eye Tracking](https://www.jspsych.org/7.3/overview/eye-tracking/) webgazer.js documentation states that: 
> The [official version of WebGazer](https://webgazer.cs.brown.edu/#download) is currently not supported by jsPsych. Our [fork of the library](https://github.com/jspsych/WebGazer) contains some minor improvements aimed at the kind of experiments that jsPsych is typically used for, e.g., situations in which the timing of display screens needs to be accurate.

As there are now several releases of Webgazer, may I check which [release of Webgazer](https://github.com/brownhci/WebGazer/releases) is the jsPsych extension based off of? Thank you.

**Comment by brkfst42** [comment]
It's Webgazer 2.0.1.  Found the answer to my own question at the [README.md](https://gitlab.pavlovia.org/demos/demo_eye_tracking2/blob/master/README.md) for demo_eye_tracking_2.

**Reply by derekalbert** [reply]
Does anyone know if it's been updated since? I'd like to use the latest version, because it seem to work better, based on the demo from the webgazer website. Does anyone know how hard it would be to update it?
